client/src/App.js

```diff
@@ -16,6 +15,7 @@ function App() {
 
   React.useEffect(() => {
     // Verify if the user is validated and if so, setUser
+    if (!user._id) user.getCurrent();
   });
```

Comment:
This function will run on every render, ideally we only want to run it on component mount. To do that with useEffect you provide an empty array as the second argument. If you want I can explain how this works.
```suggestion
  }, []);
```

Reply:
resolved in #54